server/routes/service-provider/appointment/post-session-feedback/attendance/postSessionAttendanceFeedbackView.ts

```diff
@@ -1,9 +1,9 @@
-import { TextareaArgs } from '../../../../../../utils/govukFrontendTypes'
-import ViewUtils from '../../../../../../utils/viewUtils'
-import PostSessionAttendanceFeedbackPresenter from './postSessionAttendanceFeedbackPresenter'
+import { TextareaArgs } from '../../../../../utils/govukFrontendTypes'
+import ViewUtils from '../../../../../utils/viewUtils'
+import PostSessionAttendanceFeedbackPresenterComponent from './postSessionAttendanceFeedbackPresenterComponent'
 
 export default class PostSessionAttendanceFeedbackView {
-  constructor(private readonly presenter: PostSessionAttendanceFeedbackPresenter) {}
+  constructor(private readonly presenter: PostSessionAttendanceFeedbackPresenterComponent) {}
```

Comment:
again, slightly confusing interface here, we are passing an object called a 'component' to an argument called 'presenter'

Reply:
I think rather than `...Component` I will have it as:

`PostSessionAttendanceFeedbackPresenter` for a base class agnostic to appointment type

and `ActionPlanSessionPostSessionAttendanceFeedbackPresenter ` for action plan sessions

and `InitialAssessmentPostSessionAttendanceFeedbackPresenter` for initial assessment

What do you think?

Note: 
1. The reason for the separation is because the UI text is slightly different for Initial Assessment compared with ActionPlan.
2. I am referring to the initial/supplier appointments as `Initial Assessment` in the code but as `Supplier Assessment` to the user. So any text and url will contain supplier-assessment. Do you think we should just globally refer to these appointments as supplier assessment appointments?